tests/assets_path_test.py

```diff
@@ -1,21 +1,17 @@
 #  Copyright (c) 2023 Mira Geoscience Ltd.
 #
-#  This file is part of octree_creation_app package.
+#  This file is part of octree-creation-app package.
 #
-#  All rights reserved.
-#
-
-from geoapps_utils.importing import assets_path
+#  octree-creation-app is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).
 
-import octree_creation_app
+from octree_creation_app import assets_path
 
 
 def test_assets_directory_exist():
-    assert assets_path(octree_creation_app.ASSETS_DIR).is_dir()
```

Comment:
from the application, we need an easy way to get the assets folder.
Writing is `assets_path(octree_creation_app.ASSETS_DIR)` everytime is somewhat heavy